### ISSUE TITLE
Launch at Login Problem Manjaro XFCE 

### ISSUE THREAD

**Issue by mkoseoglu** [Ulauncher/Ulauncher#127]
I have installed System Ulauncher. The "Launch at Login" box is not clicked in Prefences.
I made Ulauncher Stable.
Linux mksglu 4.9.24-1-MANJARO x86_64 GNU/Linux

**Comment by gornostal** [comment]
Hi,

Do you mean that you cannot check "Launch at Login"?
If so, could you try to close Preferences and open them again?
Let me know if that helps.

If issue still exists, please send logs `~/.cache/ulauncher_cache/last.log`

**Comment by mkoseoglu** [comment]
@gornostal I tried to restart it. Failed.
not clicked. I had to make and run it on Manjaro from "Session and Startup" settings. Log file is empty.

**Comment by gornostal** [comment]
Could you run it from terminal then?
`ulauncher -v`

Please send output.

**Comment by mkoseoglu** [comment]
It worked with this command. After entering the command I entered the settings and marked the button. I restarted my computer and it started working. 
`/usr/lib/python2.7/site-packages/ulauncher/ext/notification.py:1: PyGIWarning: Notify was imported without specifying a version first. Use gi.require_version('Notify', '0.7') before import to ensure that the right version gets loaded.
  from gi.repository import Notify
/usr/lib/python2.7/site-packages/ulauncher/ui/windows/PreferencesUlauncherDialog.py:5: PyGIWarning: WebKit2 was imported without specifying a version first. Use gi.require_version('WebKit2', '4.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gio, Gtk, WebKit2
2017-05-07 00:16:32,191:DEBUG:ulauncher: set_up_logging() 'logging enabled'
2017-05-07 00:16:32,192:INFO:ulauncher: main() 'Ulauncher version 2.2.0.r1'
2017-05-07 00:16:32,192:INFO:ulauncher: main() 'GTK+ 3.22.10'
2017-05-07 00:16:32,193:DEBUG:ulauncher: main() 'Getting the existing instance...'
`
